.github/workflows/tests.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
@kmax12 How would you feel about removing 3.8 and 3.9 from the test matrix?

Reply:
Fine by me